src/combinatorics/simple_graph/basic.lean

```diff
@@ -899,6 +899,14 @@ outside the set. This is a wrapper around `simple_graph.comap`. -/
 @[reducible] def induce (s : set V) (G : simple_graph V) : simple_graph s :=
 G.comap (function.embedding.subtype _)
 
+@[simp] lemma induce_singleton_eq_top (v : V) : G.induce {v} = ⊤ :=
```

Comment:
Isn't that a general lemma about `comap`?

Reply:
What would that lemma be? I can think of `G.induce s = \top`, but not much more general.